libcaf_test/caf/test/runner.cpp

```diff
@@ -53,6 +53,24 @@ std::optional<unsigned> parse_log_level(std::string_view x) {
   return {};
 }
 
+std::optional<std::regex> to_regex(std::string_view regex_string) {
+#ifdef CAF_ENABLE_EXCEPTIONS
+  try {
```

Comment:
To avoid redundant code in the catch blocks below:

```suggestion
#ifdef CAF_ENABLE_EXCEPTIONS
  using detail::format_to;
  auto err_out = [] {
    return std::ostream_iterator<char>{std::cerr};
  };
  try {
```